charts/bpdm/values.yaml

```diff
@@ -0,0 +1,81 @@
+bpdm-gate:
+  enabled: true
+  replicaCount: 1
+  image:
+    registry: ghcr.io
+    repository: catenax-ng/tx-bpdm/gate
+    pullPolicy: Always
+    tag: ""
+  # ... more configuration for bpdm-gate if need...
+  applicationConfig:
+    bpdm:
+      pool:
+        base-url: ""
+
+bpdm-pool:
+  enabled: true
+  replicaCount: 1
+  image:
+    registry: ghcr.io
+    repository: catenax-ng/tx-bpdm/pool
+    pullPolicy: Always
+    tag: ""
+  opensearch:
+    masterService: ""
+    fullnameOverride: "bpdm-pool-opensearch"
```

Comment:
Is opensearch naming needed here? I feel since the dependency is not created should be not needed, right?

Reply:
for now as it stands we have a need to have it like this , we will take a look further on this with the help of dev sec team